report-generator/src/main/java/com/teamscale/report/util/ILogger.java

```diff
@@ -22,6 +22,11 @@
 	/** Logs at error level. */
 	void error(Throwable throwable);
 
-	/** Logs at error level. */
+	/** Logs at error level. The given {@link Throwable} may be null. */
```

Comment:
Please also add this to the WARN method - and maybe also add another default implementation, to be consistent with the `error()` case?

Reply:
since we don't need warn(message) at the moment, I wouldn't add unused code. Rather add it when needed